internal/lsp/lsp.go

```diff
@@ -45,11 +46,18 @@ func (s *Server) Handle(ctx context.Context, conn *jsonrpc2.Conn, r *jsonrpc2.Re
 	jsonrpc2.HandlerWithError(s.handle).Handle(ctx, conn, r)
 }
 
+func logJSONPtr(msg *json.RawMessage) string {
```

Comment:
longer term: this should probably be a top-level zerolog abstraction built into our logging package